microsoft-edge: 119.0.2151.44 -> 119.0.2151.72

rhysmdnz opened the pull request:
## Description of changes

fixes the problems with https://github.com/NixOS/nixpkgs/pull/269463

## Things done

<!-- Please check what applies. Note that these are not hard requirements but merely serve as information for reviewers. -->

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [ ] `sandbox = true`
- [x] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [x] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) (or backporting [23.05 Release notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2305.section.md))
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

<!--
To help with the large amounts of pull requests, we would appreciate your
reviews of other pull requests, especially simple package updates. Just leave a
comment describing what you have tested in the relevant package/service.
Reviewing helps to reduce the average time-to-merge for everyone.
Thanks a lot if you do!

List of open PRs: https://github.com/NixOS/nixpkgs/pulls
Reviewing guidelines: https://nixos.org/manual/nixpkgs/unstable/#chap-reviewing-contributions
-->

### Priorities

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

OPNA2608 committed:
qmenumodel: init at 0.9.1

OPNA2608 committed:
qmenumodel: 0.9.1 -> 0.9.2

OPNA2608 committed:
properties-cpp: 0.0.1 -> 0.0.3

r-ryantm committed:
icingaweb2-thirdparty: 0.11.0 -> 0.12.0

pbsds committed:
micro: 2.0.12 -> 2.0.13

Diff: https://github.com/zyedidia/micro/compare/v2.0.12...v2.0.13

Also make desktop entry non-executable

r-ryantm committed:
gnuplot: 5.4.9 -> 5.4.10

r-ryantm committed:
abbaye-des-morts: 2.0.1 -> 2.0.2

jmbaur committed:
flarectl: init at 0.80.0

Flarectl is a CLI application for interacting with a Cloudflare account.

r-ryantm committed:
recyclarr: 6.0.1 -> 6.0.2

r-ryantm committed:
kubeone: 1.7.0 -> 1.7.1

eyJhb committed:
transgui: unstable-2022-02-02 -> unstable-2023-10-19

r-ryantm committed:
spire: 1.8.2 -> 1.8.3

r-ryantm committed:
suricata: 7.0.1 -> 7.0.2

r-ryantm committed:
abaddon: 0.1.12 -> 0.1.13

r-ryantm committed:
tar2ext4: 0.11.1 -> 0.11.4

r-ryantm committed:
trunk-ng: 0.17.10 -> 0.17.11

change hash
change hash (again)

r-ryantm committed:
xlockmore: 5.73 -> 5.74

pbsds committed:
pagefind: 1.0.3 -> 1.0.4

Diff: https://github.com/cloudcannon/pagefind/compare/refs/tags/v1.0.3...v1.0.4

nessdoor committed:
nixos/sourcehut: remove usages of 'with lib;'

Inherit library attributes at a finer granularity, instead.

nessdoor committed:
nixos/sourcehut: add myself as maintainer

r-ryantm committed:
antidote: 1.9.2 -> 1.9.3

r-ryantm committed:
delve: 1.21.0 -> 1.21.2

r-ryantm committed:
doomretro: 5.0.4 -> 5.0.7

r-ryantm committed:
bind: 9.18.19 -> 9.18.20

r-ryantm committed:
gomplate: 3.11.5 -> 3.11.6

r-ryantm committed:
log4cplus: 2.1.0 -> 2.1.1

DamienCassou committed:
blobfuse: 2.1.0 -> 2.1.2

r-ryantm committed:
xcp: 0.12.0 -> 0.12.1

r-ryantm committed:
goffice: 0.10.55 -> 0.10.56

r-ryantm committed:
iosevka-bin: 27.3.1 -> 27.3.5

vbgl committed:
coqPackages.VST: 2.12 → 2.13

coqPackages.compcert: 3.13 → 3.13.1

r-ryantm committed:
matio: 1.5.23 -> 1.5.26

SomeoneSerge committed:
python311Packages.torch: choose magma at the expression level

...instead of at the callPackage site.
Addresses https://github.com/NixOS/nixpkgs/issues/268736

SomeoneSerge committed:
python311Packages.torch: fix typo in the cuda&&rocm error message

SomeoneSerge committed:
magma: respect the global isStatic

r-ryantm committed:
zinit: 3.12.0 -> 3.12.1

r-ryantm committed:
peg: 0.1.18 -> 0.1.19

r-ryantm committed:
mill: 0.11.5 -> 0.11.6

uninsane committed:
mpv-unwrapped: fix shebangs for cross-compiled builds

previously umpv had the build python and mpv_identify.sh had the build
bash, even though both of these are intended to run on the host.

r-ryantm committed:
jetbrains-toolbox: 2.1.0.18144 -> 2.1.1.18388

lilyinstarlight committed:
python3Packages.{shiboken6,pyside6}: 6.5.2 -> 6.6.0

r-ryantm committed:
highlight: 4.7 -> 4.10

mweinelt committed:
ctranslate2: 3.21.0 -> 3.22.0

https://github.com/OpenNMT/CTranslate2/blob/v3.22.0/CHANGELOG.md

Commit:
amdgpu_top: 0.1.11 -> 0.3.1

marsam committed:
llpp: update homepage

marsam committed:
llpp: remove unused inputs

marsam committed:
llpp: remove unneeded configurePhase

marsam committed:
llpp: fix license

r-ryantm committed:
nordzy-icon-theme: 1.8.5 -> 1.8.6

ShamrockLee committed:
treewide: use lib.warnIf instead of improvised alternatives

dotlambda committed:
setzer: 61 -> 62

Diff: https://github.com/cvfosammmm/Setzer/compare/v61...v62

r-ryantm committed:
micronaut: 4.1.3 -> 4.1.6

r-ryantm committed:
mujs: 1.3.3 -> 1.3.4

adisbladis committed:
lib.attrsets.matchAttrs: Avoid some list allocations when walking structure

Benchmarks (`nix-instantiate ./. -A python3`):

- Before:
``` json
{
  "cpuTime": 0.29049500823020935,
  "envs": {
    "bytes": 4484216,
    "elements": 221443,
    "number": 169542
  },
  "gc": {
    "heapSize": 402915328,
    "totalBytes": 53086800
  },
  "list": {
    "bytes": 749424,
    "concats": 4242,
    "elements": 93678
  },
  "nrAvoided": 253991,
  "nrFunctionCalls": 149848,
  "nrLookups": 49612,
  "nrOpUpdateValuesCopied": 1587837,
  "nrOpUpdates": 10104,
  "nrPrimOpCalls": 130356,
  "nrThunks": 358981,
  "sets": {
    "bytes": 30423600,
    "elements": 1859999,
    "number": 41476
  },
  "sizes": {
    "Attr": 16,
    "Bindings": 16,
    "Env": 16,
    "Value": 24
  },
  "symbols": {
    "bytes": 236145,
    "number": 24453
  },
  "values": {
    "bytes": 10502520,
    "number": 437605
  }
}
```

- After:
``` json
{
  "cpuTime": 0.2946169972419739,
  "envs": {
    "bytes": 3315224,
    "elements": 172735,
    "number": 120834
  },
  "gc": {
    "heapSize": 402915328,
    "totalBytes": 48718432
  },
  "list": {
    "bytes": 347568,
    "concats": 4242,
    "elements": 43446
  },
  "nrAvoided": 173252,
  "nrFunctionCalls": 101140,
  "nrLookups": 73595,
  "nrOpUpdateValuesCopied": 1587837,
  "nrOpUpdates": 10104,
  "nrPrimOpCalls": 83067,
  "nrThunks": 304216,
  "sets": {
    "bytes": 29704096,
    "elements": 1831673,
    "number": 24833
  },
  "sizes": {
    "Attr": 16,
    "Bindings": 16,
    "Env": 16,
    "Value": 24
  },
  "symbols": {
    "bytes": 236145,
    "number": 24453
  },
  "values": {
    "bytes": 8961552,
    "number": 373398
  }
}
```

r-ryantm committed:
navidrome: 0.50.0 -> 0.50.1

r-ryantm committed:
dippi: 4.0.2 -> 4.0.6

r-ryantm committed:
neocmakelsp: 0.6.8 -> 0.6.11

r-ryantm committed:
olm: 3.2.15 -> 3.2.16

andresilva committed:
polkadot: 1.3.0 -> 1.4.0

Myaats committed:
samba: fix samba-tool

This commit addresses the following issues:
- samba-tool requiring python built with libxcrypt-legacy
- adding the missing markdown python package
- fixing the shebang patching so disallowedReferences passes

r-ryantm committed:
fldigi: 4.2.00 -> 4.2.03

r-ryantm committed:
obs-studio-plugins.obs-vertical-canvas: 1.2.5 -> 1.3.0

robbevp committed:
vscode-extensions.arrterian.nix-env-selector: 1.0.9 -> 1.0.10

fabaff committed:
metasploit: 6.3.43 -> 6.3.44

Linschlager committed:
haskellPackages.hidapi: fix darwin build

wineee committed:
wl-gammactl: don't need wlroots

h7x4 committed:
cyme: 1.5.2 -> 1.6.0

h7x4 committed:
cyme: add passthru.tests.version

r-ryantm committed:
monkeysAudio: 10.26 -> 10.28

marsam committed:
esbuild: 0.19.7 -> 0.19.8

Diff: https://github.com/evanw/esbuild/compare/v0.19.7...v0.19.8

Changelog: https://github.com/evanw/esbuild/blob/v0.19.8/CHANGELOG.md

K900 committed:
treewide: /lib/libexec -> /libexec

/lib/libexec doesn't exist.

patka-123 committed:
tox-prpl: drop

r-ryantm committed:
postgresql12JitPackages.pgrouting: 3.5.1 -> 3.6.0

marsam committed:
postgresqlPackages.pgrouting: fix build darwin

adisbladis committed:
lib.attrsets.hasAttrByPath: Don't allocate one extra list per lookup recursion

Using `tail` in a recursive loop like this needlessly allocates.
This changes the loop to look up by list index instead.

adisbladis committed:
lib.attrsets.attrByPath: Don't allocate one extra list per lookup recursion

Using `tail` in a recursive loop like this needlessly allocates.
This changes the loop to look up by list index instead.

maralorn committed:
nix-output-monitor: 2.0.0.7 -> 2.1.1

r-ryantm committed:
snd: 23.8 -> 23.9

r-ryantm committed:
checkSSLCert: 2.76.0 -> 2.77.0

amarshall committed:
zfs: default disable zfs_dmu_offset_next_sync to avoid data corruption

This helps mitigate a data corruption bug. This was previously defaulted
to zero prior to upstream commit
05b3eb6d232009db247882a39d518e7282630753, and it is already a tunable,
so doing this seems safe. Initially data corruption was thought to be
introduced with v2.2, but further upstream investigation currently
believes that changes in v2.2 only exacerbated the issue that already
existed.

A longer-term fix is likely to be
https://github.com/openzfs/zfs/pull/15571, though that is not yet
merged. The zfs_2_1 package has already backported that, so do not apply
the tunable default change there.

Positioning of `extraPatches` is to avoid merge conflicts with
https://github.com/NixOS/nixpkgs/pull/269097.

Patch is nearly identical to the [Gentoo][1] patch, but better patch
formatting.

See https://github.com/openzfs/zfs/issues/11900
See https://github.com/openzfs/zfs/issues/15526

[1]: https://gitweb.gentoo.org/repo/gentoo.git/tree/sys-fs/zfs-kmod/files/zfs-kmod-2.2.1-Disable-zfs_dmu_offset_next_sync-tunable-by-default.patch

r-ryantm committed:
opengrok: 1.12.21 -> 1.12.23

mfrischknecht committed:
kapacitor: fix build of embedded `libflux` dependency with current rust

The embedded `libflux` dependency of `kapacitor` fails to build with
more current rust tooling due to an unneeded `.clone()` call that
is promoted into an error message by a strict linting build config.

This introduces a patch that removes the offending method call.
The issue is already resolved upstream, but even the current `kapacitor`
version still depends on this specific version of `libflux` [1], and the
respective git commit contains a lot of other (automated) changes [2],
so cherry-picking the patch via a GitHub URL seems even more brittle.

[1]: https://github.com/influxdata/kapacitor/blob/v1.7.1/go.mod#L26
[2]: https://github.com/influxdata/flux/commit/e7f7023848929e16ad5bd3b41d217847bd4fd72b#diff-96572e971d9e19b54290a434debbf7db054b21c9ce19035159542756ffb8ab87

bew committed:
jo: add meta.mainProgram

dotlambda committed:
python311Packages.img2pdf: 0.5.0 -> 0.5.1

Changelog: https://gitlab.mister-muffin.de/josch/img2pdf/src/tag/0.5.1/CHANGES.rst

bobvanderlinden committed:
git-cola: 4.4.0 -> 4.4.1

dotlambda committed:
diebahn: 1.5.0 -> 2.1.0

Diff: https://gitlab.com/schmiddi-on-mobile/diebahn/-/compare/1.5.0...2.1.0

Changelog: https://gitlab.com/schmiddi-on-mobile/railway/-/blob/2.1.0/CHANGELOG.md

das-g committed:
osm2pgsql: 1.9.2 → 1.10.0

Co-Authored-By: Nikolay Korotkiy <sikmir@disroot.org>
Co-Authored-By: Ivan Mincik <ivan.mincik@gmail.com>

schmittlauch committed:
nixos/forgejo docs: correct phrasing

stepbrobd committed:
osu-lazer-bin: remove version from name

Reported-by: Lord-Valen <lord_valen@protonmail.com>

r-ryantm committed:
p2pool: 3.8 -> 3.9

SuperSandro2000 committed:
rclone: 1.64.2 -> 1.65.0

Diff: https://github.com/rclone/rclone/compare/v1.64.2...v1.65.0

Changelog: https://github.com/rclone/rclone/blob/v1.65.0/docs/content/changelog.md

SuperSandro2000 committed:
ripgrep: 13.0.0 -> 14.0.1

wackbyte committed:
vscode-extensions.bradlc.vscode-tailwindcss: 0.9.9 -> 0.11.30

Also adds a description and links.

fabaff committed:
python311Packages.construct: add changelog to meta

fabaff committed:
python311Packages.construct: add pythonImportsCheck

fabaff committed:
python311Packages.construct: refactor

- update ordering
- run nixpkgs-fmt

fabaff committed:
python311Packages.construct: add optional-dependencies

fabaff committed:
python311Packages.construct: remove unused input

fabaff committed:
python311Packages.construct: 2.10.68 -> 2.10.69

Diff: https://github.com/construct/construct/compare/refs/tags/v2.10.68...v2.10.69

Changelog: https://github.com/construct/construct/releases/tag/v2.10.69

r-ryantm committed:
pdm: 2.10.3 -> 2.10.4

wackbyte committed:
vscode-extensions.golang.go: 0.38.0 -> 0.40.0

dotlambda committed:
lisgd: 0.3.7 -> 0.4.0

Changelog: https://git.sr.ht/~mil/lisgd/refs/0.4.0

dotlambda committed:
python311Packages.imap-tools: 1.4.0 -> 1.5.0

Diff: https://github.com/ikvk/imap_tools/compare/refs/tags/v1.4.0...v1.5.0

Changelog: https://github.com/ikvk/imap_tools/blob/v1.5.0/docs/release_notes.rst

adisbladis committed:
lib.customisation: Inherit lib/builtins into scope

It makes the code more readable if we have less nested attrsets being accessed.

adisbladis committed:
lib.customisation: Don't allocate intermediate list for missing args

onny committed:
downonspot: unstable-2021-10-13 -> unstable-2023-11-26

infinisil committed:
kapacitor: fix aarch64-darwin build

r-ryantm committed:
pgrok: 1.4.0 -> 1.4.1

r-ryantm committed:
phosh: 0.32.0 -> 0.33.0

dotlambda committed:
python311Packages.recurring-ical-events: 2.1.0 -> 2.1.1

Diff: https://github.com/niccokunzmann/python-recurring-ical-events/compare/v2.1.0...v2.1.1

Changelog: https://github.com/niccokunzmann/python-recurring-ical-events/blob/v2.1.1/README.rst#changelog

r-ryantm committed:
php81Extensions.apcu: 5.1.22 -> 5.1.23

r-ryantm committed:
php81Extensions.ast: 1.1.0 -> 1.1.1

h7x4 committed:
treewide: add mainProgram

dotlambda committed:
python311Packages.bonsai: 1.5.1 -> 1.5.2

Diff: https://github.com/noirello/bonsai/compare/v1.5.1...v1.5.2

Changelog: https://github.com/noirello/bonsai/blob/v1.5.2/CHANGELOG.rst

rhysmdnz committed:
microsoft-edge: 119.0.2151.44 -> 119.0.2151.72

nessdoor committed:
nixos/sourcehut: reformat expressions using nixpkgs-fmt

r-ryantm committed:
php81Extensions.blackfire: 1.91.0 -> 1.92.0

r-ryantm committed:
php81Extensions.datadog_trace: 0.93.1 -> 0.94.0

pbsds committed:
Merge pull request #269296 from RadovanSk/amdgpu_top

amdgpu_top: 0.1.11 -> 0.3.1

pbsds committed:
Merge pull request #269288 from mweinelt/ctranslate2-3.22.0

ctranslate2: 3.21.0 -> 3.22.0

pbsds committed:
Merge pull request #269140 from r-ryantm/auto-update/mill

mill: 0.11.5 -> 0.11.6

pbsds committed:
Merge pull request #270001 from h7x4/pkgs-update-cyme

cyme: 1.5.2 -> 1.6.0, add version test

pbsds committed:
Merge pull request #267950 from pbsds/bump-pagefind-1700135765

pagefind: 1.0.3 -> 1.0.4

pbsds committed:
Merge pull request #262642 from pbsds/bump-micro-1697933115

micro: 2.0.12 -> 2.0.13

bcc32 committed:
ocamlPackages.caqti*: improve meta.license

See #269788.

kirillrdy commented on pkgs/applications/networking/browsers/microsoft-edge/default.nix:
```diff
@@ -1,20 +1,20 @@
+    version = "119.0.2151.72";
     revision = "1";
-    sha256 = "sha256-QY9Dk4tcpuNJGVcAcaIaVXAT95K87rK7ZQo7COMDpVU=";
+    sha256 = "sha256-thBx/+6thNXXKppA11IOG5EiMx7pA9FA3vSkwa9+F0o=";
```
```suggestion
    hash = "sha256-thBx/+6thNXXKppA11IOG5EiMx7pA9FA3vSkwa9+F0o=";
```

you will also need to modify ./browser.nix

kirillrdy commented on pkgs/applications/networking/browsers/microsoft-edge/default.nix:
```diff
@@ -1,20 +1,20 @@
+    version = "120.0.2210.22";
     revision = "1";
-    sha256 = "sha256-aLiitzCoMvJH2xAfo9bO7lEPMqKlb++BdJkrWx61SMc=";
+    sha256 = "sha256-GayVVZbtGLQmmXu+k4wdsD+rPwGiSPHnQwzVYyKWhHM=";
```
```suggestion
    hash = "sha256-GayVVZbtGLQmmXu+k4wdsD+rPwGiSPHnQwzVYyKWhHM=";
```

kirillrdy commented on pkgs/applications/networking/browsers/microsoft-edge/default.nix:
```diff
@@ -1,20 +1,20 @@
+    version = "121.0.2220.3";
     revision = "1";
-    sha256 = "sha256-L/rtOddk4bt8ffkRnq0BYcVjrSb7RmDaay85S5vixSM=";
+    sha256 = "sha256-M3r+SLp3lQ7oWDYoM7aNZDC5wbMxFZggsu0Iuyyw/cw=";
```
```suggestion
    hash = "sha256-M3r+SLp3lQ7oWDYoM7aNZDC5wbMxFZggsu0Iuyyw/cw=";
```

r-ryantm committed:
lucky-commit: 2.2.1 -> 2.2.2

marsam committed:
Merge pull request #270277 from SuperSandro2000/rclone

rclone: 1.64.2 -> 1.65.0

dotlambda committed:
Merge pull request #269437 from dotlambda/setzer-62

setzer: 61 -> 62

dotlambda committed:
Merge pull request #270248 from dotlambda/diebahn-2.1.0

diebahn: 1.5.0 -> 2.1.0

dotlambda committed:
Merge pull request #270243 from dotlambda/img2pdf-0.5.1

python311Packages.img2pdf: 0.5.0 -> 0.5.1

dotlambda committed:
Merge pull request #270293 from dotlambda/imap-tools-1.5.0

python311Packages.imap-tools: 1.4.0 -> 1.5.0

fabaff committed:
python311Packages.azure-mgmt-containerregistry: 10.2.0 -> 10.3.0

Changelog: https://github.com/Azure/azure-sdk-for-python/blob/azure-mgmt-containerregistry_10.3.0/sdk/containerregistry/azure-mgmt-containerregistry/CHANGELOG.md

fabaff committed:
python311Packages.azure-mgmt-datafactory: 3.1.0 -> 4.0.0

Changelog: https://github.com/Azure/azure-sdk-for-python/tree/azure-mgmt-datafactory_4.0.0/sdk/datafactory/azure-mgmt-datafactory

akgrant43 committed:
glamoroustoolkit: 1.0.2 -> 1.0.6

fabaff committed:
python311Packages.azure-mgmt-recoveryservicesbackup: 6.0.0 -> 7.0.0

Changelog: https://github.com/Azure/azure-sdk-for-python/blob/azure-mgmt-recoveryservicesbackup_7.0.0/sdk/recoveryservices/azure-mgmt-recoveryservicesbackup/CHANGELOG.md

r-ryantm committed:
python311Packages.authcaptureproxy: 1.2.1 -> 1.3.0

adisbladis committed:
Merge pull request #270133 from adisbladis/lib-attrbypath-allocs

lib.(has|)attrByPath: Don't needlessly allocate lists in loop

trofi committed:
nixVersions.unstable: fix patch name related to rapidcheck shared library

DamienCassou committed:
Merge pull request #268244 from DamienCassou/blobfuse-2.1.2

fabaff committed:
python311Packages.watermark: add optional-dependencies

pbsds committed:
Merge pull request #269643 from Myaats/samba-tool

samba: fix samba-tool

drupol committed:
Merge pull request #267212 from eyJhb/transgui-update

transgui: unstable-2022-02-02 -> unstable-2023-10-19

r-ryantm committed:
php81Packages.php-cs-fixer: 3.37.1 -> 3.40.0

drupol committed:
Merge pull request #237725 from onny/downonspot-update

downonspot: unstable-2021-10-13 -> unstable-2023-11-26

fabaff committed:
Merge pull request #270208 from r-ryantm/auto-update/checkSSLCert

checkSSLCert: 2.76.0 -> 2.77.0

fabaff committed:
Merge pull request #269842 from fabaff/metasploit-bump

metasploit: 6.3.43 -> 6.3.44

fabaff committed:
checkov: 3.1.14 -> 3.1.15

Diff: https://github.com/bridgecrewio/checkov/compare/refs/tags/3.1.14...3.1.15

Changelog: https://github.com/bridgecrewio/checkov/releases/tag/3.1.15

drupol committed:
Merge pull request #270316 from r-ryantm/auto-update/php81Extensions.ast

php81Extensions.ast: 1.1.0 -> 1.1.1

drupol committed:
Merge pull request #270314 from r-ryantm/auto-update/php81Extensions.apcu

php81Extensions.apcu: 5.1.22 -> 5.1.23

drupol committed:
Merge pull request #270330 from r-ryantm/auto-update/php81Extensions.datadog_trace

php81Extensions.datadog_trace: 0.93.1 -> 0.94.0

miallo committed:
nixos-rebuild: remove manpage hint on feature that is not merged

This is a hint that mentions the feature #105910 but that is not yet
merged. This is very confusing and since it there is no progress with
that merge we should remove the reference

marsam committed:
Merge pull request #270296 from marsam/update-esbuild

esbuild: 0.19.7 -> 0.19.8

marsam committed:
Merge pull request #270132 from r-ryantm/auto-update/postgresql12JitPackages.pgrouting

postgresql12JitPackages.pgrouting: 3.5.1 -> 3.6.0

ncfavier committed:
Merge pull request #269955 from Linschlager/hs-hidapi

haskellPackages.hidapi: fix darwin build

r-ryantm committed:
purescript: 0.15.12 -> 0.15.13

maralorn committed:
Merge pull request #270161 from maralorn/bump-nom

nix-output-monitor: 2.0.0.7 -> 2.1.1

roberth committed:
Merge pull request #270366 from h7x4/treewide-add-main-program-3

treewide: add mainProgram

mweinelt committed:
Merge pull request #270368 from r-ryantm/auto-update/python311Packages.authcaptureproxy

python311Packages.authcaptureproxy: 1.2.1 -> 1.3.0

infinisil committed:
Merge pull request #270233 from mfrischknecht/fix-kapacitor-build

kapacitor: fix build of embedded `libflux` dependency with current rust

pbsds committed:
Merge pull request #245911 from OPNA2608/init/qmenumodel

qmenumodel: init at 0.9.2

fabaff committed:
Merge pull request #270364 from fabaff/azure-mgmt-recoveryservicesbackup-bump

python311Packages.azure-mgmt-recoveryservicesbackup: 6.0.0 -> 7.0.0

fabaff committed:
Merge pull request #270362 from fabaff/azure-mgmt-datafactory-bump

python311Packages.azure-mgmt-datafactory: 3.1.0 -> 4.0.0

fabaff committed:
Merge pull request #270363 from fabaff/azure-mgmt-containerregistry-bump

python311Packages.azure-mgmt-containerregistry: 10.2.0 -> 10.3.0

zimbatm committed:
fahclient: 7.6.21 -> 8.1.18 (#246832)

Rebuild from source

lilyinstarlight committed:
Merge pull request #269757 from marsam/refactor-llpp

llpp: update homepage and license

pbsds committed:
Merge pull request #259389 from OPNA2608/update/properties-cpp

properties-cpp: 0.0.1 -> 0.0.3

adamcstephens committed:
Merge pull request #270275 from amarshall/zfs-disable-dmu-offset-by-default

zfs: default disable zfs_dmu_offset_next_sync to avoid data corruption

cdepillabout committed:
Merge pull request #270407 from r-ryantm/auto-update/purescript

purescript: 0.15.12 -> 0.15.13

roberth committed:
Merge pull request #269552 from adisbladis/lib-matchattrs-list-allocs

lib.attrsets.matchAttrs: Avoid some list allocations when walking structure

K900 committed:
Merge pull request #265987 from lilyinstarlight/upd/pyside6

python3Packages.{shiboken6,pyside6}: 6.5.2 -> 6.6.0

NickCao committed:
Merge pull request #269650 from ctron/auto-update/trunk-ng

Auto update/trunk ng

NickCao committed:
Merge pull request #270259 from stepbrobd/osu-lazer-bin

osu-lazer-bin: remove version from name

NickCao committed:
Merge pull request #270246 from bobvanderlinden/pr-git-cola-4.4.1

git-cola: 4.4.0 -> 4.4.1

NickCao committed:
Merge pull request #270256 from schmittlauch/gite-forgejo-migration-phrasing

nixos/forgejo docs: correct phrasing

NickCao committed:
Merge pull request #270263 from r-ryantm/auto-update/p2pool

p2pool: 3.8 -> 3.9

NickCao committed:
Merge pull request #270283 from wackbyte/bump-vscode-tailwind

vscode-extensions.bradlc.vscode-tailwindcss: 0.9.9 -> 0.11.30

sikmir committed:
Merge pull request #270173 from das-g/osm2pgsql

osm2pgsql: 1.9.2 → 1.10.0

NickCao committed:
Merge pull request #270319 from dotlambda/bonsai-1.5.2

python311Packages.bonsai: 1.5.1 -> 1.5.2

NickCao committed:
Merge pull request #270300 from r-ryantm/auto-update/pgrok

pgrok: 1.4.0 -> 1.4.1

NickCao committed:
Merge pull request #270288 from r-ryantm/auto-update/pdm

pdm: 2.10.3 -> 2.10.4

NickCao committed:
Merge pull request #270287 from wackbyte/bump-vscode-go

vscode-extensions.golang.go: 0.38.0 -> 0.40.0

NickCao committed:
Merge pull request #270309 from dotlambda/recurring-ical-events-2.1.1

python311Packages.recurring-ical-events: 2.1.0 -> 2.1.1

NickCao committed:
Merge pull request #270291 from dotlambda/lisgd-0.4.0

lisgd: 0.3.7 -> 0.4.0

NickCao committed:
Merge pull request #270356 from r-ryantm/auto-update/lucky-commit

lucky-commit: 2.2.1 -> 2.2.2

NickCao committed:
Merge pull request #270398 from fabaff/checkov-bump

checkov: 3.1.14 -> 3.1.15

NickCao committed:
Merge pull request #270387 from fabaff/watermark-fix

python311Packages.watermark: add optional-dependencies

NickCao committed:
Merge pull request #270369 from akgrant43/glamoroustoolkit-v106

glamoroustoolkit: 1.0.2 -> 1.0.6

NickCao committed:
Merge pull request #269906 from robbevp/vscode-extensions.arrterian.nix-env-selector-1.0.10

vscode-extensions.arrterian.nix-env-selector: 1.0.9 -> 1.0.10

infinisil committed:
Merge pull request #270377 from trofi/nixVersions.unstable-fix-patch-name

nixVersions.unstable: fix patch name related to rapidcheck shared lib…

infinisil committed:
Merge pull request #270299 from adisbladis/lib-customisation-allocs

lib.customisation: Refactor nested set access & avoid some allocations

pbsds committed:
Merge pull request #269453 from ShamrockLee/use-warn-if

treewide: use lib.warnIf instead of improvised alternatives

JulienMalka committed:
Merge pull request #270403 from miallo/fix-list-generations-manpage

nixos-rebuild: remove manpage hint on feature that is not merged

NickCao committed:
Merge pull request #270058 from K900/fix-libexec

treewide: /lib/libexec -> /libexec

pbsds committed:
Merge pull request #270235 from bew/patch-1

jo: add meta.mainProgram

pbsds committed:
Merge pull request #264090 from jmbaur/flarectl-init

flarectl: init at 0.80.0

yuyuyureka committed:
fastnetmon-advanced: 2.0.353 -> 2.0.356 (#268122)

infinisil committed:
slack: 4.34.121 -> 4.35.126

Commit:
Merge pull request #268746 from SomeoneSerge/fix/torch-choose-args

python311Packages.torch: choose magma at the expression level

amaxine committed:
Merge pull request #270440 from tweag/update/slack

slack: 4.34.121 -> 4.35.126, fixes blank screen rendering issue

andresilva committed:
Merge pull request #269634 from andresilva/polkadot-1.4.0

polkadot: 1.3.0 -> 1.4.0

mfrischknecht committed:
nix-ld: mark as broken on 32 bit targets (#269919)

See https://github.com/Mic92/nix-ld/issues/64

drupol committed:
Merge pull request #270327 from r-ryantm/auto-update/php81Extensions.blackfire

php81Extensions.blackfire: 1.91.0 -> 1.92.0

drupol committed:
Merge pull request #269281 from r-ryantm/auto-update/highlight

highlight: 4.7 -> 4.10

drupol committed:
Merge pull request #263154 from r-ryantm/auto-update/abbaye-des-morts

abbaye-des-morts: 2.0.1 -> 2.0.2

drupol committed:
Merge pull request #262709 from r-ryantm/auto-update/gnuplot

gnuplot: 5.4.9 -> 5.4.10

drupol committed:
Merge pull request #259536 from r-ryantm/auto-update/icingaweb2-thirdparty

icingaweb2-thirdparty: 0.11.0 -> 0.12.0

drupol committed:
Merge pull request #270394 from r-ryantm/auto-update/php81Packages.php-cs-fixer

php81Packages.php-cs-fixer: 3.37.1 -> 3.40.0

drupol committed:
Merge pull request #269815 from r-ryantm/auto-update/obs-studio-plugins.obs-vertical-canvas

obs-studio-plugins.obs-vertical-canvas: 1.2.5 -> 1.3.0

drupol committed:
Merge pull request #269549 from r-ryantm/auto-update/mujs

mujs: 1.3.3 -> 1.3.4

drupol committed:
Merge pull request #269606 from r-ryantm/auto-update/olm

olm: 3.2.15 -> 3.2.16

drupol committed:
Merge pull request #269175 from r-ryantm/auto-update/jetbrains-toolbox

jetbrains-toolbox: 2.1.0.18144 -> 2.1.1.18388

drupol committed:
Merge pull request #268568 from r-ryantm/auto-update/iosevka-bin

iosevka-bin: 27.3.1 -> 27.3.5

drupol committed:
Merge pull request #269116 from r-ryantm/auto-update/peg

peg: 0.1.18 -> 0.1.19

drupol committed:
Merge pull request #267380 from r-ryantm/auto-update/abaddon

abaddon: 0.1.12 -> 0.1.13

drupol committed:
Merge pull request #264149 from r-ryantm/auto-update/matio

matio: 1.5.23 -> 1.5.26

drupol committed:
Merge pull request #268185 from r-ryantm/auto-update/bind

bind: 9.18.19 -> 9.18.20

drupol committed:
Merge pull request #268012 from r-ryantm/auto-update/antidote

antidote: 1.9.2 -> 1.9.3

drupol committed:
Merge pull request #267374 from r-ryantm/auto-update/suricata

suricata: 7.0.1 -> 7.0.2

drupol committed:
Merge pull request #267855 from r-ryantm/auto-update/xlockmore

xlockmore: 5.73 -> 5.74

drupol committed:
Merge pull request #268266 from r-ryantm/auto-update/xcp

xcp: 0.12.0 -> 0.12.1

drupol committed:
Merge pull request #268068 from r-ryantm/auto-update/delve

delve: 1.21.0 -> 1.21.2

drupol committed:
Merge pull request #262193 from r-ryantm/auto-update/tar2ext4

tar2ext4: 0.11.1 -> 0.11.4

drupol committed:
Merge pull request #267344 from r-ryantm/auto-update/spire

spire: 1.8.2 -> 1.8.3

drupol committed:
Merge pull request #267152 from r-ryantm/auto-update/kubeone

kubeone: 1.7.0 -> 1.7.1

drupol committed:
Merge pull request #268120 from r-ryantm/auto-update/doomretro

doomretro: 5.0.4 -> 5.0.7

drupol committed:
Merge pull request #270216 from r-ryantm/auto-update/opengrok

opengrok: 1.12.21 -> 1.12.23

drupol committed:
Merge pull request #270190 from r-ryantm/auto-update/snd

snd: 23.8 -> 23.9

drupol committed:
Merge pull request #270023 from r-ryantm/auto-update/monkeysAudio

monkeysAudio: 10.26 -> 10.28

drupol committed:
Merge pull request #269807 from r-ryantm/auto-update/fldigi

fldigi: 4.2.00 -> 4.2.03

drupol committed:
Merge pull request #264921 from r-ryantm/auto-update/micronaut

micronaut: 4.1.3 -> 4.1.6

drupol committed:
Merge pull request #269561 from r-ryantm/auto-update/navidrome

navidrome: 0.50.0 -> 0.50.1

drupol committed:
Merge pull request #269568 from r-ryantm/auto-update/dippi

dippi: 4.0.2 -> 4.0.6

drupol committed:
Merge pull request #269572 from r-ryantm/auto-update/neocmakelsp

neocmakelsp: 0.6.8 -> 0.6.11

drupol committed:
Merge pull request #268989 from r-ryantm/auto-update/zinit

zinit: 3.12.0 -> 3.12.1

drupol committed:
Merge pull request #267039 from r-ryantm/auto-update/recyclarr

recyclarr: 6.0.1 -> 6.0.2

drupol committed:
Merge pull request #268527 from r-ryantm/auto-update/goffice

goffice: 0.10.55 -> 0.10.56

drupol committed:
Merge pull request #268236 from r-ryantm/auto-update/log4cplus

log4cplus: 2.1.0 -> 2.1.1

drupol committed:
Merge pull request #268222 from r-ryantm/auto-update/gomplate

gomplate: 3.11.5 -> 3.11.6

drupol committed:
Merge pull request #269380 from r-ryantm/auto-update/nordzy-icon-theme

nordzy-icon-theme: 1.8.5 -> 1.8.6

kirillrdy commented:
this pings too many people, feel free to open new PR

rhysmdnz commented:
Done in #270479 not sure what happened there

rhysmdnz commented:
Hmmm, my running theory is it compared to an old copy of master as part of https://www.githubstatus.com/incidents/66vhjmd266r9 and so picked up extra commits as part of the pr
